<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videoWindow</class>
 <widget class="QMainWindow" name="videoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
QPushButton#btn_play[mode='pause']{border-image:url(:/image/player/play.svg);}
QPushButton#btn_play[mode='play']{border-image:url(:/image/player/pause.svg);}</string>
   </property>
   <widget class="QPushButton" name="btn_prev">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>550</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/player/next.svg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="play_progress_bar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>961</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListView" name="lyricView">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>761</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView {
        background: rgba(255,255,255,0.1);
        border-radius: 12px;
        padding: 20px 0;
        font-family: &quot;Microsoft YaHei&quot;;
      font-size: 20px;
        outline: none;
    }
    
    QListView::item {
        height: 50px;
qproperty-alignment: AlignCenter;
        color: white;
        border: none;
        text-align: center;
    }
    
    QListView::item:selected {
        color: #e74c3c;
        background: rgba(231, 76, 60, 0.1);
        font-size: 24px;
        font-weight: bold;
        border-radius: 8px;
    }
    
    QScrollBar:vertical {
        width: 8px;
        background: rgba(200, 200, 200, 0.1);
    }
    
    QScrollBar::handle:vertical {
        background: rgba(231, 76, 60, 0.6);
        border-radius: 4px;
    }</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_voice">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>560</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_play">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>540</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_file">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>550</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/player/file.svg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_del">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab{height:30;width:100}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_local_3">
     <attribute name="title">
      <string>本地列表</string>
     </attribute>
     <widget class="QListWidget" name="list_local_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_online_3">
     <attribute name="title">
      <string>在线列表</string>
     </attribute>
     <widget class="QListWidget" name="list_online_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_quit">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>550</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/player/back.svg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_next">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>550</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/player/prev.svg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>751</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="video_Layout"/>
   </widget>
   <widget class="QPushButton" name="btn_fullscreen">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>545</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/fullscreen.svg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_speed">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>588</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>车速：0 km/h</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
